File: Java ejs/UD3/UD03A032.java
import java.util.Scanner;
public class UD03A032{
	public static void main(String[] args){
		int ang1,ang2,ang3;
        Scanner teclado=new Scanner(System.in);
        System.out.println("Introduzca un ángulo: ");
        ang1=teclado.nextInt();
        System.out.println("Introduzca un ángulo: ");
        ang2=teclado.nextInt();
        System.out.println("Introduzca un ángulo: ");
        ang3=teclado.nextInt();
        if (ang1+ang2+ang3==180)
        {
            if (ang1==90 || ang2==90 || ang3==90)
            {
                System.out.println("Es un triángulo rectángulo");
            }else{
                if (ang1>90 || ang2>90 || ang3>90)
                {
                    System.out.println("Es un triángulo obtusángulo");
                }else{
                    System.out.println("Es un triángulo acutángulo");
                }
            }
        }
	}
}

File: Python ejs/UD03/A03/UD03A032.py
ang1=int(input("Introduzca ángulo: "))
ang2=int(input("Introduzca ángulo: "))
ang3=int(input("Introduzca ángulo: "))

if ang1+ang2+ang3==180:
    if ang1==90 or ang2==90 or ang3==90:
        print("Es un triángulo rectángulo")
    elif ang1>90 or ang2>90 or ang3>90:
        print("Es un triángulo obtusángulo")
    else:
        print("Es un triángulo acutángulo")
